.templates/template_location.json

```diff
@@ -11,6 +11,12 @@
     "templatePath": "python-guestbook",
     "templateName": "Python: Guestbook"
   },
+  {
+    "repoPath": "https://github.com/GoogleCloudPlatform/cloud-code-samples.git",
```

Comment:
Adding this will add this immediately to the the Cloud Code extensions once checked it.  Are you okay to hold off on this for a bit so we can do a little more testing?

Reply:
Sure. Thanks for the reminder.